packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -17,22 +17,18 @@
 import classNames from "classnames";
 import * as React from "react";
 
-import { AbstractPureComponent, DISPLAYNAME_PREFIX, type IntentProps } from "../../common";
+import { AbstractPureComponent, DISPLAYNAME_PREFIX, type IntentProps, Utils } from "../../common";
 import * as Classes from "../../common/classes";
+import * as Errors from "../../common/errors";
 // eslint-disable-next-line import/no-cycle
 import { Popover, type PopoverInteractionKind } from "../popover/popover";
 import { TOOLTIP_ARROW_SVG_SIZE } from "../popover/popoverArrow";
 import type { DefaultPopoverTargetHTMLProps, PopoverSharedProps } from "../popover/popoverSharedProps";
 import { TooltipContext, type TooltipContextState, TooltipProvider } from "../popover/tooltipContext";
 
 export interface TooltipProps<TProps extends DefaultPopoverTargetHTMLProps = DefaultPopoverTargetHTMLProps>
-    extends Omit<PopoverSharedProps<TProps>, "shouldReturnFocusOnClose">,
+    extends Omit<PopoverSharedProps<TProps, { tooltipId: string }>, "shouldReturnFocusOnClose">,
         IntentProps {
-    /**
-     * The content that will be displayed inside of the tooltip.
-     */
-    content: React.JSX.Element | string;
```

Comment:
same comment as https://github.com/palantir/blueprint/pull/6875/files#r1655441543

Reply:
I think you commented this on an old diff. I reverted the change to this type.